site/content/blog/2023-09-15-browser-recorder/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "GSoC 2023 Browser Recorder"
+summary: "ZAP has introduced a new feature to record pre-task activities such as logging in etc. using Browser Recorder."
+type: post
+tags:
+- blog
+- browser recorder
+- client
+- GSoC 2023
+date: "2023-09-15"
+authors:
+    - aryangupta701
```

Comment:
Do you need to add your author details in this PR?

Reply:
no